google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
I need to read more about the arguments to form a full opinion here, but there seems to be a prevailing school of thought that says that using Optionals for method arguments is bad practice.

Regarding the overloaded method - if I do it here, you'll notice that I may have to do it again in `ProjectRepositoriesModelItem#loadRepositories` too. It may make sense, but my initial feeling was that its cleaner to allow @Nullable arguments. Ill consider it more.